tsfile/src/main/java/org/apache/iotdb/tsfile/read/reader/LocalTsFileInput.java

```diff
@@ -145,4 +146,13 @@ public void close() throws IOException {
   public int readInt() {
     throw new UnsupportedOperationException();
   }
+
+  @Override
+  public String readVarIntString(long offset) throws IOException {
```

Comment:
this method... is not thread-safe... see how other functions use the channel.

Reply:
Ok, I've changed it to be thread-safe.